documentation/docs/how_tos/accounts_and_addresses/03_request_funds.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Request Funds
+description: 'How to programmatically request testnet funds.'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- faucet
+- testnet
+- test funds
+- nodejs
+- python
+- rust
+---
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from "../../_admonitions/_guide_coming_soon.md";
+import NodejsOutput from "../../_admonitions/_guide_coming_soon.md";
+import PythonCode from "../../_admonitions/_guide_coming_soon.md";
+import PythonOutput from "../../_admonitions/_guide_coming_soon.md";
+import RustCode from "../../libraries/rust/how_to/accounts_and_addresses/_03_request_funds/_code_sample.mdx";
+import RustOutput from "../../libraries/rust/how_to/accounts_and_addresses/_03_request_funds/_expected_output.md";
+import TipFaucet from "../../_admonitions/_tip_faucet.md";
+
+Once you have [created your account](01_create_a_wallet_account.mdx) and [generated an address](02_generate_address.mdx),
+you can start using wallet.rs for value transactions.  Since these examples target the testnet, you can request funds
```

Comment:
```suggestion
Once you have [created your account](01_create_a_wallet_account.mdx),
you can start using wallet.rs for value transactions. Since these examples target the testnet, you can request funds
```
02_generate_address is not necessary, since we already got an address in account creation